### PR TITLE
Add a reference that describes all keys and certs used in Corda 5

### PR DESCRIPTION
**Pull request by nadinequinr3**
Preview: https://niamh.preview.docs.r3.com/en/platform/corda/5.1/reference/certificates.html

### CODE REVIEW

File: content/en/platform/corda/5.1/reference/certificates.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+# Corda Keys and Certificates
+
+This section describes the different types of keys and certificates used by Corda 5. It contains the following:
+
+{{< toc >}}
+
+## Application Keys
+
+Corda uses the following types of keys:
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.1/reference/certificates.md:21]:
```suggestion
Corda uses the following types of key:
```

File: content/en/platform/corda/5.1/reference/certificates.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+| Master Wrapping    | Decrypts, through wrapping keys, private keys. This master wrapping key is not available in the database but in higher trust zones (crypto worker and secret managers, for example) that enable decrypting database-encrypted wrapping keys. Encrypted wrapping keys can then decrypt the database-encrypted `VNodePrivateKey`. The crypto processor configuration specifies a set of master wrapping keys as SALT and passphrase parameters for key derivation. This produces a symmetric key which is used for wrapping (encrypting at rest) all corda managed wrapping keys. For more information, see [Key Wrapping]({{< relref "../key-concepts/cluster-admin/_index.md#key-wrapping">}}).                                                                                                                                                                                                                                                                                                                                                           | AES-256 symmetric master key                         |
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.1/reference/certificates.md:39]:
```suggestion
| Master Wrapping    | Decrypts, through wrapping keys, private keys. This master wrapping key is not available in the database but in higher trust zones (for example, crypto worker and secret managers) that enable decrypting database-encrypted wrapping keys. Encrypted wrapping keys can then decrypt the database-encrypted `VNodePrivateKey`. The crypto processor configuration specifies a set of master wrapping keys as SALT and passphrase parameters for key derivation. This produces a symmetric key which is used for wrapping (encrypting at rest) all Corda-managed wrapping keys. For more information, see [Key Wrapping]({{< relref "../key-concepts/cluster-admin/_index.md#key-wrapping">}}).                                                                                                                                                                                                                                                                                                                                                           | AES-256 symmetric master key                         |
```

File: content/en/platform/corda/5.1/reference/certificates.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+| Sensitive Config   | Decrypts sensitive values encrypted at rest (for example, in the database or Kafka). This key is derived from the `Sensitive config SALT and PASSPHRASE` [credential](#credentials).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | AES-256 symmetric data encryption key                |
+
+## Infrastructure Keys
+
+Corda uses the following types of infrastructure keys:
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.1/reference/certificates.md:48]:
```suggestion
Corda uses the following types of infrastructure key:
```

File: content/en/platform/corda/5.1/reference/certificates.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+| Client Kafka credentials for a specific worker | Kafka credentials (username and password) created at deployment time and stored by default in Kubernetes etcd as secret configuration. This value is passed to worker pods memory at runtime.                                              | High entropy unencrypted string       |
+| Kubernetes secrets creation via API            | The Cluster Adminstrator creates the secrets in Kubernetes etcd to be used by the Corda helm charts. This is executed optionally when Helm charts overrides are not used as an alternative.                                                |                                       |
+| Kubernetes secrets configured in Helm charts   | The Cluster Adminstrator configures the secrets as overrides in helm charts. This is an alternative to creating Kubernetes secrets via API.                                                                                                                     |                                       |
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.1/reference/certificates.md:79]:
```suggestion
| Kubernetes Secrets configured in Helm charts   | The Cluster Administrator configures the Secrets as overrides in Helm charts. This is an alternative to creating Kubernetes Secrets via API.                                                                                                                     |                                       |
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.1/reference/certificates.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+| CPB Signing      | Code-signing key for publishing CPIs. The CorDapp publisher uses this key to sign a {{< tooltip >}}CPB{{< /tooltip >}} before sending it to the Network Operator to use when [creating the CPI]({{< relref "../application-networks/creating/members/cpi.md#import-code-signing-certificates" >}}). For more information, see [Code Signing]({{< relref "../developing-applications/packaging/code-signing/_index.md">}}).                                                                   | 256-bit X.509 public signature verification key |
+| CPI Publisher    | Code-signing key for publishing CPIs. Corda validates that uploaded {{< tooltip >}}CPIs{{< /tooltip >}} are signed with a trusted key. The Network Operator usually [imports the signing key to Corda]({{< relref "../application-networks/creating/members/cpi.md#import-code-signing-certificates" >}}). In some cases, such as the Notary CPI, R3 manages the key. For more information, see [Code Signing]({{< relref "../developing-applications/packaging/code-signing/_index.md">}}). | 256-bit X.509 public signature verification key | 256-bit X.509 public signature verification key
+
+## PKI assets
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.1/reference/certificates.md:58]:
```suggestion
## PKI Assets
```

File: content/en/platform/corda/5.1/reference/certificates.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+# Corda Keys and Certificates
+
+This section describes the different types of keys and certificates used by Corda 5. It contains the following:
+
+{{< toc >}}
+
+## Application Keys
+
+Corda uses the following types of key:
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.1/reference/certificates.md:21]:
```suggestion
Corda uses the following types of application key:
```

File: content/en/platform/corda/5.1/reference/certificates.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+| CPK Signing      | Code-signing key for publishing CPKs. The CorDapp Developer uses this key to sign a {{< tooltip >}}CPK{{< /tooltip >}} before bundling it into a CPB. For more information, see [Code Signing]({{< relref "../developing-applications/packaging/code-signing/_index.md">}}).                                                                                                                                                                                                                 | 256-bit X.509 public signature verification key |
+| CPB Signing      | Code-signing key for publishing CPIs. The CorDapp publisher uses this key to sign a {{< tooltip >}}CPB{{< /tooltip >}} before sending it to the Network Operator to use when [creating the CPI]({{< relref "../application-networks/creating/members/cpi.md#import-code-signing-certificates" >}}). For more information, see [Code Signing]({{< relref "../developing-applications/packaging/code-signing/_index.md">}}).                                                                   | 256-bit X.509 public signature verification key |
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.1/reference/certificates.md:55]:
```suggestion
| CPB Signing      | Code-signing key for publishing CPIs. The CorDapp publisher uses this key to sign a {{< tooltip >}}CPB{{< /tooltip >}} before sending it to the Network Operator to use when [creating the CPI]({{< relref "../application-networks/creating/members/cpi.md#create-the-cpi-file" >}}). For more information, see [Code Signing]({{< relref "../developing-applications/packaging/code-signing/_index.md">}}).                                                                   | 256-bit X.509 public signature verification key |
```

File: content/en/platform/corda/5.1/reference/certificates.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+| SSO Config credentials                         | SSO credentials used by the HTTP Gateway process. This is an API ID provided by Azure AD.                                                                                                                                                  | String                                |
+| Installation credentials creation              | Installation credentials with enough entropy created in Kubernetes etcd, Helm Chart, or another vault. These credentials are used for key derivation and cluster database secret encryption.                                                 | Passphrase and salt                   |
+| Client Kafka credentials for a specific worker | Kafka credentials (username and password) created at deployment time and stored by default in Kubernetes etcd as secret configuration. This value is passed to worker pods memory at runtime.                                              | High entropy unencrypted string       |
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.1/reference/certificates.md:77]:
```suggestion
| Client Kafka credentials for a specific worker | Kafka credentials (user name and password) created at deployment time and stored by default in Kubernetes etcd as secret configuration. This value is passed to worker pods memory at runtime.                                              | High entropy unencrypted string       |
```

File: content/en/platform/corda/5.1/reference/certificates.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+| Kubernetes secrets creation via API            | The Cluster Adminstrator creates the secrets in Kubernetes etcd to be used by the Corda Helm charts. This is executed optionally when Helm Chart overrides are not used as an alternative.                                                |                                       |
+| Kubernetes Secrets configured in Helm charts   | The Cluster Administrator configures the Secrets as overrides in Helm charts. This is an alternative to creating Kubernetes Secrets via API.                                                                                                                     |                                       |
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.1/reference/certificates.md:79]:
```suggestion
| Kubernetes Secrets configured in Helm charts   | The Cluster Administrator configures the secrets as overrides in Helm charts. This is an alternative to creating Kubernetes secrets via API.                                                                                                                     |                                       |
```

File: content/en/platform/corda/5.1/reference/certificates.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+| Kafka Truststore               | Kafka truststore for server authentication (TLS).                                                                                                                                                      | JKS               |
+| Session Keys X.509 Certificate | Session keys certificate validated during session initialization. For more information, see [Session Certificates]({{< relref "../application-networks/creating/optional/session-certificates.md" >}}). | X.509 Certificate |
+| E2E Identity Trust Roots       | Trust root of certificates for session initiation keys.                                                                                                                                                | 256-bit PEM       |
+| Certificate for Corda releases | Trusted certificates for binary Corda release verification.                                                                                                                                            | X.509 Certificate |
+
+## Credentials
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.1/reference/certificates.md:69]: Some credentials in this table do not have a type provided. I assumed it's on purpose.

File: content/en/platform/corda/5.1/reference/certificates.md (hunk 9 above)
Comment (nadinequinr3) [reply content/en/platform/corda/5.1/reference/certificates.md:79]: Capitalised in the Kubernetes docs so leaving as is.

**Review by AnnaK-R3 — APPROVED** [review]

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**